### PR TITLE
Fix validate bids

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_validate_bids.py [hunk 1]
```diff
@@ -200,8 +201,12 @@ def get_data(layout, nSub, dwis, t1s, fs, default_readout, clean):
                                TotalReadoutTime=totalreadout,
                                invalid_filters='drop')
 
-    related_files = [curr_related for curr_related in related_files if curr_related.entities['suffix'] != 'dwi']
+    related_files_filtered = []
+    for curr_related in related_files:
+        if curr_related.entities['suffix'] != 'dwi' and curr_related.entities['extension'] == '.nii.gz':
```
Comment (AlexVCaron) [inline scripts/scil_validate_bids.py:206]:
```suggestion
        if curr_related.entities['suffix'] != 'dwi' and \
            curr_related.entities['extension'] == '.nii.gz':
```

**Review by AlexVCaron — COMMENTED** [review]
Everything runs well on my end. There is a few lines of code going over 80 characters, could you look into that ? Else, LGTM !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**